### PR TITLE
Add authSource & ssl options to MongoDB connector fix

### PR DESCRIPTION
**Pull request by mtringuyen**
- Updated strapi-generate-new package with ssl option
- Updated strapi-mongoose package allow connect with ssl option

### CODE REVIEW

File: packages/strapi-mongoose/lib/index.js [hunk 1]
```diff
@@ -50,6 +51,7 @@ module.exports = function (strapi) {
             connectOptions.pass = password
           }
         }
+        connectOptions.ssl = ssl ? true : false;
```
Comment (lauriejim) [inline packages/strapi-mongoose/lib/index.js]: `connectOptions.ssl = ssl` is the same nop ?

Comment (mtringuyen) [reply packages/strapi-mongoose/lib/index.js]: yes, ssl is a boolean. but this is for ssl is undefined

File: packages/strapi-generate-new/files/config/environments/production/database.json [hunk 2]
```diff
@@ -10,7 +10,8 @@
         "port": "${process.env.DATABASE_PORT || 27017}",
         "database": "${process.env.DATABASE_NAME || 'production'}",
         "username": "${process.env.DATABASE_USERNAME || ''}",
-        "password": "${process.env.DATABASE_PASSWORD || ''}"
+        "password": "${process.env.DATABASE_PASSWORD || ''}",
+        "ssl": false
```
Comment (lauriejim) [inline packages/strapi-generate-new/files/config/environments/production/database.json]: Maybe it's could be a cool to also have an environment variable for this setting ?

**Comment by lauriejim** [comment]
I merge this one and continu the discussion on this PR https://github.com/strapi/strapi/pull/753

**Pull request merged**